project_national/national_upgrades.yml

```diff
@@ -44,20 +44,46 @@ baseline:
   n_buildings_represented: 110000000
 
 upgrades:
-  - upgrade_name: Windows and Walls
+  - upgrade_name: Windows
     options:
       - option: Windows|Triple, Low-E, Non-metal, Air, L-Gain
         costs:
           - value: 45.77
             multiplier: Window Area (ft^2)
         lifetime: 30
+
+  - upgrade_name: Walls
+    options:
       - option: Insulation Wall|Wood Stud, R-13
-        apply_logic: Insulation Wall|Wood Stud, Uninsulated
         costs:
           - value: 2.21
             multiplier: Wall Area, Above-Grade, Conditioned (ft^2)
         lifetime: 999
 
+  - upgrade_name: Air Leakage
+    options:
+      - option: Infiltration|3 ACH50
+        costs:
+          - value: 3.55
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 999
+
+  - upgrade_name: Refrigerator
+    options:
+      - option: Refrigerator|EF 21.9, 100% Usage
+        costs:
+          - value: 2500.0
+            multiplier: Fixed (1)
+        lifetime: 17.4
+
+  - upgrade_name: Lighting
```

Comment:
Can we add a few more upgrades? I'm thinking HVAC (ASHP?), water heater (HPWH?), and PV.